### PR TITLE
Upgrade javacpp version to 1.5.6

### PR DESCRIPTION
**Pull request by Berstanio**
This fixes https://github.com/oracle/graal/issues/3684
Most of the details about this pr can be found in the linked issue.

I build javacpp and the javacpp llvm preset for use in svm with the newest javacpp version and uploaded it on a private server. But it should be uploaded to "https://lafo.ssw.uni-linz.ac.at/pub/graal-external-deps/native-image" when this should get merged. If needed the packages can also be build easily from source. Just checkout this branch "https://github.com/Berstanio/javacpp/tree/graal-llvm-javacpp" and run "mvn install". Then checkout this "https://github.com/Berstanio/javacpp-presets/tree/graal-llvm-preset" and also run "mvn install -Djavacpp.platform=[linux-x86_64 | linux-arm64 | macosx-x86_64]" (on the necessary host machines).

### CODE REVIEW

**Comment by Berstanio** [comment]
The signed oca is now approved.

File: substratevm/mx.substratevm/suite.py [hunk 1]
```diff
@@ -70,36 +70,36 @@
...
         },
         "JAVACPP_SHADOWED": {
-            "sha1" : "212aaddcd73448c7b6da781fb6cde934c667dc2c",
-            "sourceSha1" : "3e9cfc02750ba8ea3babc1b8546a50ec36b849a2",
-            "urlbase": "https://lafo.ssw.uni-linz.ac.at/pub/graal-external-deps/native-image",
-            "urls": ["{urlbase}/javacpp-shadowed-1.5.2.jar"],
-            "sourceUrls": ["{urlbase}/javacpp-shadowed-1.5.2-sources.jar"],
+            "sha1" : "761db712863a8ac533ef4e28ddcfdd49bc71955c",
+            "sourceSha1" : "da52e2a8b39415f279e388a0e68d77817d471ae3",
+            "urlbase": "http://mos.bplaced.net/llvm",
```
Comment (Noisyfox) [inline substratevm/mx.substratevm/suite.py:85]: Should we use the jars from maven central?
https://repo1.maven.org/maven2/org/bytedeco/llvm/12.0.1-1.5.6/

Comment (Noisyfox) [reply substratevm/mx.substratevm/suite.py:85]: Ah I guess the one from maven central is not shadowed?

Comment (Berstanio) [reply substratevm/mx.substratevm/suite.py:85]: I don't know why it was done in the first place, maybe there were some llvm patches that got applied. I gave it a quick test with the maven packages and it worked fine. Maybe someone of the graal team can comment, why the prebuild packages weren't used for JavaCPP 1.5.2?

**Pull request closed without merging**